### ISSUE TITLE
Remove time and date from 4x1 widget

### ISSUE THREAD

**Issue by robinpaulson** [martykan/forecastie#374]
As explained in #348 , both are probably redundant.

**Comment by igor-cali** [comment]
The date is not redundant, as it is not part of the standard Android status bar and screen.
In my opinion, also the time should be kept in the widget; as a user I can get at a glance all the relevant info (date, time and weather).

If this issue aims to provide a simpler 4x1 widget, my recommendation is to make it as additional to the current one.

**Comment by azakosath** [comment]
I will agree that time is redundant, it would be nice if it could be replaced with a timestamp instead and a refresh action, so you could easily update the current conditions without entering the app itself.

**Comment by robinpaulson** [comment]
@azakosath When you say "timestamp", do you mean the last-updated time, as on the 5x2 widget?

**Comment by azakosath** [comment]
@robinpaulson, yes that's what I meant. The truth is that I hadn't ever noticed that the last-updated time was there on the 4x2 widget, because I'm using transparent widgets over a dark wallpaper. Maybe you should consider changing the colour too.

**Comment by robinpaulson** [comment]
There is an issue concerning widget text colour.

**Comment by robinpaulson** [comment]
#355
